src/reference/sectionA_definitions.inc

```diff
@@ -119,7 +119,7 @@ Specific information items
 
    #. A document type declaration information item;
 
-   #. An unparsed entity information item;
+   #. An unparsed entity information item; or
```

Comment:
Moving these `or`s around will be some nice needless bookkeeping :roll_eyes:

Reply:
I've gotta keep a job somehow ...